frontend/src/routes/_protected/treecluster/_formular/$treeclusterId.edit.tsx

```diff
@@ -41,8 +44,9 @@ function EditTreeCluster() {
         replace: true,
       })
       showToast('Die Bewässerungsgruppe wurde erfolgreich editiert.')
+      queryClient.invalidateQueries(treeClusterIdQuery(clusterId))
```

Comment:
The query client caches the response from the backend for a period of time. Here I invalidate the caches after each update and create so that we can guarantee that the data on the next screen is up to date.